src/m365/teams/commands/user/user-app-add.ts

```diff
@@ -10,7 +12,8 @@ interface CommandArgs {
 }
 
 interface Options extends GlobalOptions {
-  id: string;
+  id?: string;
```

Comment:
Since we've changed the ID to be optional, we need to update its validator so that it doesn't fail when the ID is not specified.